NAA_Analysis.py

```diff
@@ -0,0 +1,245 @@
+from __future__ import print_function
+from SpectrumFileBase import SpectrumFileBase
+import SPEFile
+import numpy as np
+import peakutils as p
+import os
+import pandas as pd
+import Gamma_Analysis as ga
+import radionuclide_scraper as rs
+import Isotopic_Abudance as ia
+
+
+def peak_finder(measurement):
+    E0 = measurement.energy_cal[0]
+    Eslope = measurement.energy_cal[1]
+    energy_axis = E0 + Eslope*measurement.channel
+    increment = 25
+    counts = measurement.data
+    peaks_found = []
+    plop = []
+    start = 0
+    end = start + increment
+    count = 0
+    for energy in energy_axis:
+        count += 1
+        if end >= len(measurement.channel):
+            break
+        else:
+            count_total = sum(counts[start:end])
+            average_range = count_total / (len(counts[start:end]))
+            average_ends = (counts[start] + counts[end]) / 2
+            threshold = 1.1 * average_ends
+            if average_range > threshold:
+                counts_range = counts[start:end]
+                middle = int((start + end) / 2)
+                average_fwhm = 0.05 * energy_axis[middle]**0.5
+                average_fwhm_dist = int(average_fwhm / Eslope)
+                indexes = p.indexes(np.log(counts_range),
+                                    thres=0.1, min_dist=average_fwhm_dist)
+                for index in indexes:
+                    plop.append(index+start)
+                    uncertainty = average_range**0.5
+                    significance = ((counts[start+index]-average_range) /
+                                    uncertainty)
+                    if significance > 4:
+                        peaks_found.append(energy_axis[index + start])
+                    else:
+                        pass
+                start += increment
+                end = start + increment
+            else:
+                start += 1
+                end = start + increment
+    return peaks_found
+
+
+def count_rate(M, B, energy):
+    """
+    Takes in a measured and background spectra and a peak energy and return the
+    net area under the peak.
+    """
+    pm_results = ga.peak_measurement(M, energy, sub_regions='none')
+    bm_results = ga.peak_measurement(B, energy, sub_regions='none')
+    sub_peak = ga.background_subtract(pm_results, bm_results, M.livetime,
+                                      B.livetime)
+    net_area = sub_peak[0]
+    count_rate = net_area/M.livetime
+
+    return(count_rate)
+
+
+def isotope_verifier(isotope):
+    """
+    Takes in a list of isotopes and checks to see whether or not the isotope is
+    naturally occuring or not. If the isotope is naturally occuring, then the
+    module will display the isotope along with its mass number and natural
+    abundance.
```

Comment:
"If the isotope is naturally occuring, then the module will display the isotope..." that is not something that this function does. Please revise the docstring.

Also, "Takes in a list of isotopes..." but it looks to me like it operates on just a single isotope.

In addition to a sentence or short paragraph, a useful docstring would contain something like:
```
Input:
  isotope: an Isotope object to check
Output:
  (flag, mass_number, isotopic_abundance, cross_section)
  flag: True if the isotope is naturally occuring
  mass_number: an integer
  isotopic_abundance: (atom fraction or mass fraction?)
  cross_section: (what cross section exactly, and what units?)
```